NuGet.config

```diff
@@ -17,6 +17,6 @@
     </config>
   <packageSources>
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
-    <add key="MyGetApplicationInsights" value="https://www.myget.org/F/applicationinsights/api/v3/index.json" />
+    <add key="MyGetApplicationInsights" value="https://www.myget.org/F/applicationinsights/" />
```

Comment:
We should update the NuGet exe's and move everything to v3, not downgrade to v2, because v2 it is going away and we will be broken when it does.  Also, there is some new functionality in v3 that is required in the new .NET Core world (and probably will be elsewhere) with regards to NuGet packages that we produce (I don't recall the details off-hand).